src/upgrades/pre_0_10_0/cas_migration.rs

```diff
@@ -77,192 +81,79 @@ impl CasMigration {
             new_agg_store,
             ca_objects_migration,
         }
-        .prepare_new_data(mode)
+        .upgrade(mode)
     }
 }
 
-impl UpgradeStore for CasMigration {
-    fn prepare_new_data(&self, mode: UpgradeMode) -> Result<(), PrepareUpgradeError> {
-        // check existing version, wipe if needed
-        self.preparation_store_prepare()?;
+impl UpgradeAggregateStorePre0_14 for CasMigration {
```

Comment:
Errm, I'm confused. This is called `Pre0_14` but is inside the `pre_0_10_0` directory? What am I missing?

Reply:
Well, it's both.. but the trait is generic for all pre0_14 migrations.